src/main/java/seedu/mark/logic/commands/TabCommand.java

```diff
@@ -0,0 +1,54 @@
+package seedu.mark.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.mark.commons.core.index.Index;
+import seedu.mark.logic.commands.commandresult.CommandResult;
+import seedu.mark.logic.commands.commandresult.TabCommandResult;
+import seedu.mark.model.Model;
+
+public class TabCommand extends Command {
+
+    public static final String COMMAND_WORD = "tab";
+
+    public static final String MESSAGE_SWITCH_ACKNOWLEDGEMENT = "Switching view as requested.";
+
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD + ": Switches view to the tab identified by the index"
+            + "given. Dashboard tab is 1, Online tab is 2, Offline tab is 3.\n"
+            + "Parameters: INDEX (must be either 1, 2 or 3)\n"
```

Comment:
I think we don't need to mention `1, 2 or 3` again! If users type wrongly, they'll get an informative error message that will help them to correct their command.
```suggestion
            + "Parameters: INDEX\n"
```